Server/routes/Web/itemCategory.js

```diff
@@ -5,12 +5,6 @@ const itemCategoryCtrl = require('../../controllers/Common/itemCategory');
 const jwtMiddleware = require('../../middleware/jwt');
 
 router.get("/", async function (req, res, next) {
-  try {
```

Comment:
Why removing it ?

Reply:
I removed the loader because otherwise, every time the user typed in the input field, it triggered a request. However, the problem was that the loader caused the entire page to reload. So, I removed it because otherwise, the user would have to search letter by letter and click on the field each time, which is not very practical.